lib/Constants.ts

```diff
@@ -1,2 +1,3 @@
 export const JIRA_ISSUE_PATTERN: RegExp = /[A-Z][A-Z0-9]*-\d+/g;
+// Use https://sonarsource-sandbox-608.atlassian.net for testing
```

Comment:
Note for future generations an their dev branches

Reply:
Can you make this info public?

Reply:
What do you mean?

Reply:
I was wondering if it is ok to have our sandbox url in a public repository.

Reply:
Ok, I can remove it. The instance itself is publicly accessible, with all its data. So there should not be an issue